src/universal/components/SocketHealthMonitor.js

```diff
@@ -39,45 +47,18 @@ class SocketHealthMonitor extends Component {
       )
     }
   }
-  onConnected = (payload) => {
-    const {dispatch} = this.props
-    const {version} = payload
-    const versionInStorage = window.localStorage.getItem(APP_VERSION_KEY)
-    if (version !== versionInStorage) {
-      popUpgradeAppToast({dispatch})
-    }
-  }
   onDisconnected = () => {
-    const {
-      atmosphere: {subscriptionClient},
-      dispatch
-    } = this.props
-    if (!subscriptionClient.reconnecting) {
+    const {dispatch} = this.props
+    this.disconnectedToastTimer = setTimeout(() => {
+      this.disconnectedToastTimer = undefined
       dispatch(
         showWarning({
-          autoDismiss: 10,
+          autoDismiss: 5,
           title: 'You’re offline!',
           message: 'We’re trying to reconnect you'
         })
       )
-    }
-  }
-
-  onSocketsDisabled = () => {
-    const {dispatch} = this.props
-    raven.captureBreadcrumb({
-      category: 'network',
-      level: 'error',
-      message: 'WebSockets Disabled'
-    })
-    dispatch(
-      showError({
-        autoDismiss: 0,
-        title: 'WebSockets Disabled',
-        message: `We weren't able to create a live connection to our server. 
-          Ask your network administrator to enable WebSockets.`
-      })
-    )
+    }, 1000)
```

Comment:
if the firewall disconnects things pretty regularly, we don't want to alert the client, so we only send alerts when they've been disconnected for more than 1 second